android/envoy/src/main/kotlin/org/greatfire/envoy/UrlUtil.kt

```diff
@@ -149,5 +149,63 @@ class UrlUtil {
 
             return sanitizedString
         }
+
+        @JvmStatic
+        fun getServiceFromUrl(url: String): String? {
+            if (url.isNotEmpty()) {
+                val split = url.split("://")[0]
+                Utils.d(TAG, "getServiceFromUrl split: $split")
+                return split
+            }
+            return null
+        }
+
+        @JvmStatic
+        fun getSanitizedUrlList(urls: List<String>): ArrayList<String> {
+            val sanitizedUrls = ArrayList<String>()
+            for (url in urls)
+                sanitizedUrls += urls.joinToString(separator = ",", transform = { sanitizeUrl(it).take(30) })
```

Comment:
i don't think that the truncation is necessary now that the urls are being shortened by being sanitized.  the longest output i could find came to 34 characters.

Reply:
ok, removed